shanoir-uploader/src/main/java/org/shanoir/uploader/ShUpConfig.java

```diff
@@ -18,9 +18,9 @@ public class ShUpConfig {
 	/**
 	 * Constants
 	 */
-	public static final String SHANOIR_UPLOADER_VERSION = "v9.0.0";
+	public static final String SHANOIR_UPLOADER_VERSION;
 	
-	public static final String RELEASE_DATE = "2025-04-29";
+	public static final String RELEASE_DATE = "2025-06-27"; // Do we keep it ?
```

Comment:
Hi @LaurentPV, I think we can remove this? We will know the release date via Github and the artifacts produced? With kind regards, Michael